README.md

```diff
@@ -4,80 +4,59 @@ minideb
 [![Build Status](https://travis-ci.org/bitnami/minideb.svg?branch=master)](https://travis-ci.org/bitnami/minideb)
 [![Slack](http://slack.oss.bitnami.com/badge.svg)](http://slack.oss.bitnami.com)
 
-A small base image designed for use in containers. The image
-is based on glibc for wide compatibility, and has apt for
-access to a large number of packages. The image is based
-on Debian, with some things that aren't required in containers
-removed:
-
-  * Some packages that aren't often used in containers
-    (hardware related, init systems etc.)
-  * Some files that aren't usually required (docs, man pages,
-    locales, caches)
-
-This image aims to strike a good balance between having
-small images, and having many quality packages available
-for easy integration.
-
-These images also include an `install_packages` command
-that you can use instead of apt. This takes care of some things
-for you:
-
-  1. Install the named packages, skipping prompts etc.
-  2. Clean up the apt metadata afterwards to keep the image small.
-  3. Retrying if apt fails. Sometimes a package will fail to download
-     due to a network issue, and this may fix that, which is
-     particularly useful in an automated build pipeline.
-
-e.g.
-
+# What is Minideb
+A small Debian based image designed for use in containers.
```

Comment:
A minimalist Debian based image built specifically to be used as a base image for containers.